daisy/step_deprecate_images.go

```diff
@@ -76,10 +83,16 @@ func (d *DeprecateImages) run(ctx context.Context, s *Step) DError {
 		wg.Add(1)
 		go func(di *DeprecateImage) {
 			defer wg.Done()
-
-			w.LogStepInfo(s.name, "DeprecateImages", "%q --> %q.", di.Image, di.DeprecationStatus.State)
-			if err := w.ComputeClient.DeprecateImage(di.Project, di.Image, &di.DeprecationStatus); err != nil {
-				e <- newErr("failed to deprecate images", err)
+			if di.DeprecationStatusAlpha.State != "" {
```

Comment:
I assume the reason to call deprecate alpha is because Alpha API uses the rollout features of the image  while GA doesn't? Otherwise there's no difference in DeprecationStatus between the two.

Reply:
Correct, the rollout feature, [StateOverride *RolloutPolicy](https://pkg.go.dev/google.golang.org/api@v0.44.0/compute/v0.alpha#DeprecationStatus.StateOverride), is only available in the Alpha API.